lib/chingu/classic_game_object.rb

```diff
@@ -187,11 +192,12 @@ def alpha=(value)
     # Sets angle, normalize it to between 0..360
     #
     def angle=(value)
-      if value < 0
-        value = 360+value
+      if value.negative?
+        value = 360 + value
       elsif value > 360
-        value = value-360
+        value -= 360
       end
+
       @angle = value
```

Comment:
```suggestion
      @angle = Gosu.normalize_angle(value)
```
...and delete all the lines above this one. Gosu.normalize_angle is slightly better because it can also normalizes values outside of the range -360..360, e.g. -370 -> 350.